email-confirmation/functions/index.js

```diff
@@ -53,5 +55,7 @@ exports.sendEmailConfirmation = functions.database.ref('/users/{uid}').onWrite(e
   mailOptions.text = 'I hereby confirm that I will stop sending you the newsletter.';
   return mailTransport.sendMail(mailOptions).then(() => {
     console.log('New unsubscription confirmation email sent to:', val.email);
+  }).catch(error => {
+    console.log(error);  
```

Comment:
just do `console.error('There was an error while sending the email:', error)` here :)